src/main/webapp/config.css

```diff
@@ -0,0 +1,84 @@
+.hidden {
+    display: none;
+}
+
+.text-align-right {
```

Comment:
Oh the joys of a backend dev applying styling names... 😛  When you create class names in css the names should reflect the content or function of the HTML element, rather than the visual style being applied. Makes it way easier to search in your IDE for class names and find what you're looking for.

An example - the `margin-bottom-2em` class name looks like it's being used to wrap the connected sites section so you could call it something like `connected-sites-container`